internal/glrepo/resolver.go

```diff
@@ -134,6 +130,7 @@ func (r *ResolvedRemotes) BaseRepo(prompt bool) (Interface, error) {
 	if remote == nil {
 		remote = r.remotes[0]
 		resolution, _ = FullNameFromURL(selectedRepo.HTTPURLToRepo)
+		resolution = "base:" + resolution
```

Comment:
This is a part that breaks @profclems

Reply:
I'm thinking we can get the older `glab-resolved=base` if set and rename it

Reply:
From what I understand these resolutions (the ones that are not `base`  or `head`) are made because there is no matching remote.

I think we can avoid breakage if for `BaseRepo` we check for `glab-resolved = base` and the first `glab-resolved` that does not have the value of `head:`

Resolutions made without the prefix continue to work that way.

Reply:
so if `strings.HasPrefix(r.Resolved, "base:")`  fails we fallback on trying `!strings.hasPrefix(r.Resolved, "head:")` which will match those before

Reply:
See: https://github.com/profclems/glab/pull/414/files#r549038702